hermes_core/__init__.py

```diff
@@ -24,4 +24,4 @@
 INST_TO_SHORTNAME = dict(zip(INST_NAMES, INST_SHORTNAMES))
 INST_TO_TARGETNAME = dict(zip(INST_NAMES, INST_TARGETNAMES))
 
-log.info(f"hermes_core version: {__version__}")
+# log.info(f"hermes_core version: {__version__}")
```

Comment:
I was noticed this when testing:
![top-before-bottom-after](https://user-images.githubusercontent.com/19628176/206756477-933bd38a-dd8b-40b8-a133-9a97bec6086b.png)
Was the comment out on purpose?